Optionally Skip `progvar` Tests

FranklandJack opened the pull request:
* Optionally skip the proram scope globa variable tests on OpenCL-3.0
devices that optionally do not support programscope global variables.

gwawiork commented on test_conformance/basic/test_progvar.cpp:
```diff
@@ -1537,6 +1572,18 @@ int test_progvar_prog_scope_misc(cl_device_id device, cl_context context, cl_com
+    }
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
```
Skipping progvar_prog_scope_uninit

FranklandJack replied:
Fixed.

gwawiork commented on test_conformance/basic/test_progvar.cpp:
```diff
@@ -1553,6 +1600,18 @@ int test_progvar_prog_scope_uninit(cl_device_id device, cl_context context, cl_c
+    }
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
```
Skipping progvar_prog_scope_init

FranklandJack replied:
Fixed.

gwawiork commented on test_conformance/basic/test_progvar.cpp:
```diff
@@ -1570,6 +1629,18 @@ int test_progvar_prog_scope_init(cl_device_id device, cl_context context, cl_com
+    }
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
```
Skipping progvar_func_scope

FranklandJack replied:
Fixed.

gwawiork commented on test_conformance/basic/test_progvar.cpp:
```diff
@@ -1518,6 +1541,18 @@ static int l_user_type( cl_device_id device, cl_context context, cl_command_queu
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
+                 "supported on this device");
```
Please add `\n` in the end of `log_info` message

FranklandJack replied:
Fixed

gwawiork commented on test_conformance/basic/test_progvar.cpp:
```diff
@@ -1537,6 +1572,18 @@ int test_progvar_prog_scope_misc(cl_device_id device, cl_context context, cl_com
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
+                 "supported on this device");
```
Please add `\n` in the end of `log_info` message

gwawiork commented on test_conformance/basic/test_progvar.cpp:
```diff
@@ -1553,6 +1600,18 @@ int test_progvar_prog_scope_uninit(cl_device_id device, cl_context context, cl_c
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
+                 "supported on this device");
```
Please add `\n` in the end of `log_info` message

FranklandJack replied:
Fixed.

gwawiork commented on test_conformance/basic/test_progvar.cpp:
```diff
@@ -1570,6 +1629,18 @@ int test_progvar_prog_scope_init(cl_device_id device, cl_context context, cl_com
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
+                 "supported on this device");
```
Please add `\n` in the end of `log_info` message

FranklandJack replied:
Fixed.

FranklandJack commented:
Thanks for your feedback @gwawiork. Good catch! I have changed the names of the functions in the log messages and added newline characters as you suggested.
